buildenv/docker/mkdocker.sh

```diff
@@ -422,6 +422,12 @@ fi
   echo " && ln -sf g++ /usr/bin/c++ \\"
   echo " && rm -f gcc-7.tar.xz"
   echo ""
+  echo "# Install gcc-11.2."
+  echo "RUN cd /usr/local \\"
+  echo " && $wget_O gcc-11.tar.xz 'https://ci.adoptopenjdk.net/userContent/gcc/gcc112.$arch.tar.xz' \\"
+  echo " && tar -xJf gcc-11.tar.xz \\"
+  echo " && rm -f gcc-11.tar.xz"
+  echo ""
```

Comment:
This seems reasonable. Have you tested the result?

Reply:
Yes, looks good. After this is merged I'll build a new container on dockerhub via https://openj9-jenkins.osuosl.org/view/Infrastructure/job/Build-Jenkins-Agent-Container/ and then modify the jdk8,11 builds to use it with gcc 11.2.
The change will be similar to https://github.com/eclipse-openj9/openj9/compare/master...pshipton:openj9:xadopt but using `eclipseopenj9/jenkins-agent-x86-centos6` rather than `adoptopenjdk/centos6_build_image`.

```
docker run -it localhost/openj9/cent6 /bin/bash
[jenkins@7703735b8486 /]$ /usr/local/gcc11/bin/gcc-11.2 --version
gcc-11.2 (Adoptium/sxa) 11.2.0
Copyright (C) 2021 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.

[jenkins@7703735b8486 /]$ /usr/local/gcc11/bin/g++-11.2 --version
g++-11.2 (Adoptium/sxa) 11.2.0
Copyright (C) 2021 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
```